addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -875,6 +909,7 @@ export class Rtc extends Record {
                 session.isTalking = false;
             }
         }
+        this.cleanups.splice(0).forEach((cleanup) => cleanup());
```

Comment:
Why this `splice(0)`?

Reply:
it remove elements from the array, I could reset it the end too, no preference

Reply:
it removes elements from the array, I can reset it next line too I have no preference

Reply:
Ah yes, that's fine then. Not used to this syntax.